File: programmers/python/입문100/5.4_배열뒤집기★/5-4.py
# 1. 리스트슬라이싱

def solution(num_list):
    return num_list[::-1]

# 2. reversed() 함수

def solution(num_list):
    return list(reversed(num_list))

# 3. reverse() 메서드

def solution(num_list):
    num_list.reverse()
    return num_list

File: programmers/java/기초124/5.4_원소들의곱과합/5_4.java
class Solution {
    public int solution(int[] num_list) {
        // int answer = 0;
        // return answer;
        //////////////////////////////////////
        long sum = 0;
        long prod = 1;

        for (int x : num_list) {
            sum += x;
            prod *= x;
        }

        return prod < sum * sum ? 1 : 0;
        //////////////////////////////////////
    }
}